test/ModuleInterface/loading-remarks.swift

```diff
@@ -49,8 +49,10 @@ import SwiftDependency
 import SwiftNonResilientDependency
 import DirectMixedDependency
 
+// CHECK: remark: transitive dependency on 'SwiftShims' for 'Swift' is required
```

Comment:
Thoughts on:
```
'Swift' has <type> transitive dependency on 'SwiftShims'
```
? The current spelling seems a little confusing as to which is the thing depending on what (to me anyway :P).

Reply:
Good suggestion, I'll use that wording instead.